src/App.tsx

```diff
@@ -28,129 +77,66 @@ export const App: React.FC = () => {
-              <input
-                type="text"
-                className="todo__title-field"
-                placeholder="Empty todo will be deleted"
-                value="Todo is being edited now"
-              />
-            </form>
-
-            <div className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-
-          {/* This todo is in loadind state */}
-          <div className="todo">
-            <label className="todo__status-label">
-              <input type="checkbox" className="todo__status" />
-            </label>
-
-            <span className="todo__title">Todo is being saved now</span>
-            <button type="button" className="todo__remove">×</button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
-
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer">
-          <span className="todo-count">
-            3 items left
-          </span>
-
-          {/* Active filter should have a 'selected' class */}
-          <nav className="filter">
-            <a href="#/" className="filter__link selected">
-              All
-            </a>
-
-            <a href="#/active" className="filter__link">
-              Active
-            </a>
-
-            <a href="#/completed" className="filter__link">
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        <TodoList todos={visibleTodos} />
+
+        {todos.length > 0 && (
+          <footer className="todoapp__footer">
+            <span className="todo-count">
+              {`${todos.length} items left`}
+            </span>
+
+            <nav className="filter">
+              <a
+                href="#/"
+                className={cn('filter__link', {
+                  selected: filterBy === FilterBy.All,
+                })}
+                onClick={() => setFilterBy(FilterBy.All)}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={cn('filter__link', {
+                  selected: filterBy === FilterBy.Active,
+                })}
+                onClick={() => setFilterBy(FilterBy.Active)}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={cn('filter__link', {
+                  selected: filterBy === FilterBy.Completed,
```

Comment:
Consider using e.preventDefault() inside the onClick event handler to prevent the default behavior of the anchor tag when it is clicked.